app/models/categories_film.rb

```diff
@@ -0,0 +1,4 @@
+class CategoriesFilm < ActiveRecord::Base
+belongs_to :film
```

Comment:
Don't forget to indent. Run a formatter / something like rubocop over your code if that helps.